examples/text-generation/README.md

```diff
@@ -95,6 +95,22 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --max_new_tokens 100
 ```
 
+You can also run Llama2-70B on Gaudi2 with all optimizations enabled using the following command:
+```bash
+python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
+--model_name_or_path meta-llama/Llama-2-70b-hf \
+--max_new_tokens 4096 \
+--bf16 \
+--use_hpu_graphs \
+--use_kv_cache \
+--batch_size 52 \
+--attn_softmax_bf16 \
+--limit_hpu_graphs \
+--reuse_cache \
+--trim_logits
+
```

Comment:
We have to remove this blank line, otherwise the command is automatically launched when copy-pasting it.

Reply:
good catch, thanks. I have fixed it